tests/unit/resource/CMakeLists.txt

```diff
@@ -16,7 +16,9 @@ set(tests
     suspend_thread_timed
 )
 
-if (${HPX_CXX_STANDARD} GREATER 11)
+# TODO: Enable on all platforms and standards when C++14 is required and test is
+# fully fixed.
+if (NOT MSVC AND "${HPX_CXX_STANDARD}" GREATER 11)
```

Comment:
This one has a few fixes now. I think we can go with the one that was in #4293 (i.e. remove this change). I'm not sure why the diff still shows master with the old comparison though...